<launch>

    <rosparam command="load" file="$(find sensors)/config/imu0_calibration-2019-07-05_130310.yaml" ns="imu0/calibration" />
    <rosparam command="load" file="$(find sensors)/config/imu1_calibration-2019-06-05_192717.yaml" ns="imu1/calibration" />
    
    <node pkg="sensors" name="imu_viewer" type="imu_viewer" >
        <param name="imu_topic" value="/arduino/imu0" />
        <param name="origin_frame" value="odom" />
        <param name="imu_frame" value="imu_link" />
    </node>

</launch>
